README.md

```diff
@@ -1,2 +1,6 @@
 # Wand-Saber
 A novel by Caroline and Preston Zhang, combining their loves for Harry Potter and Star Wars. Enjoy!
+WELCOME... to Wand Saber ! We had the idea of Wand Saber when we were going out for a walk.We thought it 
+would be nise to write a story so we decided we would mix up STAR WARS and HARRY POTTER .We decided we would 
```

Comment:
- [ ] nise -> nice
- [ ] Add a space before "We decided". In general, add a space after every period.